src/vivarium_gates_mncnh/components/mortality.py

```diff
@@ -133,28 +123,20 @@ def on_time_step(self, event) -> None:
                 "life_expectancy"
             ](dead_idx)
 
-        # Update YLDs for each maternal disorder
-        yld_idx = has_maternal_disorders.index.difference(dead_idx)
-        for cause in self.maternal_disorders:
-            pop.loc[yld_idx, f"{cause}_ylds"] = self.lookup_tables[f"{cause}_yld_rate"](
-                yld_idx
-            )
-
         self.population_view.update(pop)
 
     ##################
     # Helper methods #
     ##################
 
-    def load_cfr_data(self, builder: Builder, key_name: str) -> pd.DataFrame:
+    def load_cfr_data(self, builder: Builder, cause: str) -> pd.DataFrame:
         """Load case fatality rate data for maternal disorders."""
-        maternal_disorder = key_name.split("_case_fatality_rate")[0]
```

Comment:
The name update is because this was incorrectly doing nothing and was not necessary.